beacon/sync/src/main/java/tech/pegasys/teku/beacon/sync/forward/multipeer/chains/ThrottlingSyncSource.java

```diff
@@ -58,7 +66,7 @@ public SafeFuture<Void> requestBlocksByRange(
       return delegate.requestBlocksByRange(startSlot, count, listener);
     } else {
       return asyncRunner.runAfterDelay(
-          () -> requestBlocksByRange(startSlot, count, listener), Duration.ofSeconds(3));
+          () -> requestBlocksByRange(startSlot, count, listener), Duration.ofSeconds(DELAY));
```

Comment:
Maybe put whole `Duration.ofSeconds(3)` to constant?

Reply:
Done, moved it to `Constants`